Description: Removes the module-info.java files to work around a bug with
 the maven-javadoc-plugin which attempts to invoke javadoc with the --module-path
 parameter if these files are found. This breaks the build when using OpenJDK 8.
 This patch can be removed once the default JDK is upgraded (or the plugin fixed).
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/imap/src/main/java/module-info.java
+++ /dev/null
@@ -1,27 +0,0 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-module com.sun.mail.imap {
-    exports com.sun.mail.iap;
-    exports com.sun.mail.imap;
-    exports com.sun.mail.imap.protocol;
-    provides javax.mail.Provider with
-	com.sun.mail.imap.IMAPProvider, com.sun.mail.imap.IMAPSSLProvider;
-
-    requires jakarta.mail;
-    requires java.logging;
-    requires java.security.sasl;
-}
--- a/mailapi/src/main/java/module-info.java
+++ /dev/null
@@ -1,33 +0,0 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-module jakarta.mail {
-    exports javax.mail;
-    exports javax.mail.event;
-    exports javax.mail.internet;
-    exports javax.mail.search;
-    exports javax.mail.util;
-    exports com.sun.mail.util;
-    exports com.sun.mail.auth;
-    exports com.sun.mail.handlers;
-
-    requires transitive jakarta.activation;
-    requires java.logging;
-    requires java.xml;		// for text/xml handler
-    requires java.desktop;	// for image/jpeg handler
-    requires java.security.sasl; // for OAuth2 support
-    uses javax.mail.Provider;
-}
--- a/mailhandler/src/main/java/module-info.java
+++ /dev/null
@@ -1,22 +0,0 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-module com.sun.mail.util.logging {
-    exports com.sun.mail.util.logging;
-
-    requires jakarta.mail;
-    requires java.logging;
-}
--- a/pop3/src/main/java/module-info.java
+++ /dev/null
@@ -1,25 +0,0 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-module com.sun.mail.pop3 {
-    exports com.sun.mail.pop3;
-    provides javax.mail.Provider with
-	com.sun.mail.pop3.POP3Provider, com.sun.mail.pop3.POP3SSLProvider;
-
-    requires jakarta.mail;
-    requires java.logging;
-    requires java.security.sasl;
-}
--- a/smtp/src/main/java/module-info.java
+++ /dev/null
@@ -1,25 +0,0 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-module com.sun.mail.smtp {
-    exports com.sun.mail.smtp;
-    provides javax.mail.Provider with
-	com.sun.mail.smtp.SMTPProvider, com.sun.mail.smtp.SMTPSSLProvider;
-
-    requires jakarta.mail;
-    requires java.logging;
-    requires java.security.sasl;
-}
--- a/gimap/src/main/java/module-info.java
+++ /dev/null
@@ -1,27 +0,0 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-module com.sun.mail.gimap {
-    exports com.sun.mail.gimap;
-    exports com.sun.mail.gimap.protocol;
-    provides javax.mail.Provider with
-	com.sun.mail.gimap.GmailProvider, com.sun.mail.gimap.GmailSSLProvider;
-
-    requires jakarta.mail;
-    requires java.logging;
-    requires java.security.sasl;
-    requires com.sun.mail.imap;
-}
--- a/pom.xml
+++ b/pom.xml
@@ -281,18 +281,6 @@
 				</configuration>
 			    </execution>
 			    <execution>
-				<id>module-info-compile</id>
-				<goals>
-				    <goal>compile</goal>
-				</goals>
-				<configuration>
-				    <release>9</release>
-				    <includes>
-					<include>module-info.java</include>
-				    </includes>
-				</configuration>
-			    </execution>
-			    <execution>
 				<id>default-testCompile</id>
 				<configuration>
 				    <release>9</release>
